build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/extractor/GoCommand.java

```diff
@@ -41,12 +41,14 @@
     ArtifactoryClientBuilderBase clientBuilder;
     Path path;
     String moduleName;
+    String buildInfoModuleId;
```

Comment:
Question - Why do we need both the *moduleName* and the *buildInfoModuleId* fields? I can see that *buildInfoModuleId* is the customised one, but can't we store the customised one in *moduleName*? Why do we need to store both?
Also related to this - it looks lie we call *parseModuleName()* even if we get a customised module name. Is this necessary?

Reply:
"module name" is go figure which is the unique ID of the module in the "go world", we are specify it in the go.mod, and use it to publish and import.
We cannot force the buildInfoModuleId in case the user want a different one from the name was specified in the project. (For example if he wants to aggregate several pkgs under one module, it will be impossible to import them and use them as dependencies.)